support-services/src/test/scala/com/gu/support/redemption/corporate/SetCodeStatusITSpec.scala

```diff
@@ -18,7 +18,7 @@ class SetCodeStatusITSpec extends AsyncFlatSpec with Matchers {
 
   // this is one test because it depends on external state which may not be in a particular state
   "setCodeStatus" should "set a code available and used" in {
-    val mutableCode: RedemptionCode = RedemptionCode("ITTEST-MUTABLE").right.get
+    val mutableCode: RedemptionCode = RedemptionCode("ittest-mutable").right.get
```

Comment:
will need to update this script and clean the DEV dyamo table and rereun the script 
https://github.com/guardian/support-frontend/blob/8e9954cb2b2c11345cfe4ff778da972b99e03702/support-services/src/test/scala/com/gu/support/redemption/corporate/IT_testdata.sh#L16-L40